### ISSUE TITLE
need to access Google drive and authorization

### ISSUE THREAD

**Issue by HimanshuN** [ivmartel/dwv#544]
can you tell me how to authorize DWV to access Google Drive and whats the need of it ?

Do I also need it for using google "client.js " and "api.js " or it is used for something else also.??

I would also like to know if I can use it for commercial purpose ???

Please reply...

**Comment by ivmartel** [comment]
The idea of the google drive integration is to directly open a dicom file from your drive. You right click on one or multiple dicom files, select open with > dwv and your file(s) open in dwv. If it is the first time, the dwv app should ask you read permission of your data.

You do not need any extra code, dwv-jqmobile should provide everything.

Now I say this but I just realized that the app does not work, I created an issue [dwv-jqmobile/issues/12](https://github.com/ivmartel/dwv-jqmobile/issues/12) to fix it.

Regarding the commercial use, I answered on #543.
